tests/cypress/e2e/features/masks_basics.js

```diff
@@ -249,7 +282,11 @@ context('Manipulations with masks', { scrollBehavior: false }, () => {
             }]];
 
             cy.startMaskDrawing();
-            cy.get('.cvat-brush-tools-underlying-pixels').click();
+            cy.get('.cvat-brush-tools-underlying-pixels').then(($el) => {
+                if (!$el.hasClass('cvat-brush-tools-active-tool')) {
+                    cy.wrap($el).click();
+                }
+            });
```

Comment:
Why do we need this change?

Reply:
Remove underlying pixels is already active here, because tool was used in previous test(which I added) and doesn't need activated again.

I thought about it and decided that it is better to [disable the tool after use](https://github.com/opencv/cvat/pull/7684/files#diff-3ea7d811d178d965da8d6b7cbc317cfe839ba9fd5c14bfb3b4f99eb4fd455710R193-R196)